lib/Devel/NYTProf/js/jit/jit.js

```diff
@@ -204,7 +204,7 @@ Class.Mutators = {
 
     Implements: function(self, klasses){
         $each($splat(klasses), function(klass){
-            Class.prototying = klass;
+            Class.prototyping = klass;
```

Comment:
This appears to be from mootools, and the typo is present in forks. This code appears to be gone from current versions...